docs/index.html

```diff
@@ -1,8 +1,141 @@
+            code span.va { color: #19177c; } /* Variable */
+            code span.vs { color: #4070a0; } /* VerbatimString */
+            code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
+        </style>
+        <style>.kommentar {display: none;}</style>
+                                               
+    </head>
+
+    <body>
+
+
+        <div class="uk-container uk-container-center uk-margin-top uk-margin-large-bottom">
+
+                        <div class="uk-grid" data-uk-grid-margin>
+                <div class="uk-width-1-1">
+                    <h1 class="uk-heading-large">u-root</h1>
+                                                        </div>
+            </div>
+            
+            <div class="uk-grid" data-uk-grid-margin >          
+                <div class="uk-width-medium-1-4">
+                    <div class="uk-overflow-container" data-uk-sticky="{top:25,media: 768}">
+                        <div class="uk-panel uk-panel-box menu-begin" >
+
+                                                        <ul>
+                                                        <li><a href="#u-root">u-root</a>
+                                                        <ul>
+                                                        <li><a href="#setup">Setup</a></li>
+                                                        <li><a href="#submitting-changes">Submitting Changes</a></li>
+                                                        <li><a href="#faqs">FAQs</a>
+                                                        <ul>
+                                                        <li><a href="#so-why-u-root">So, why “u-root”?</a></li>
+                                                        <li><a href="#any-publications">Any publications?</a></li>
+                                                        </ul></li>
+                                                        <li><a href="#community">Community</a></li>
+                                                        <li><a href="#contributors">Contributors</a></li>
+                                                        <li><a href="#logo">Logo</a></li>
+                                                        </ul></li>
+                                                        </ul>
+                            
+                        </div>
+                    </div>
+                </div>
+
+                <div class="uk-width-medium-3-4">
+<p><img src="img/u-root-logo.png" width="300" alt="u-root logo" /><br />
+</p>
 <h1 id="u-root">u-root</h1>
-<p>u-root is an embeddable root file system intended to be placed in a flash device as part of the firmware image, along with a Linux kernel. Unlike most embedded root file systems, which consist of large binaries, u-root only has five: an init program and four Go compiler binaries.</p>
+<p>u-root embodies four different projects.</p>
```

Comment:
Embodies it is not seem the best word to use it here. Maybe includes? incorporates?

Reply:
Yea, this comes right from the root README; this PR just picks its `Description` section in favor of the previous section that diverged and turned untrue. Another PR for the README would be the way to go, I think.

Reply:
see https://github.com/u-root/u-root#description